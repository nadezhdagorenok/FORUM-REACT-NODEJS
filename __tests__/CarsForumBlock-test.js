"use strict";

import React from 'react';
import renderer from 'react-test-renderer';

import CarsForumBlock from '../components/CarsForumBlock.js';

test('render CarsForumBlock', () => {

    const component = renderer.create(
        <CarsForumBlock title="CAR's Topics!" />
    );
    let componentTree = component.toJSON();
    expect(componentTree).toMatchSnapshot();
});



