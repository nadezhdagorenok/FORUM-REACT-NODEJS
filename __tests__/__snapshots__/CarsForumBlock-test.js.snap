// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render CarsForumBlock 1`] = `
<div
  className="preloader"
>
  <div
    className="loader"
  />
</div>
`;
